bin/varnishd/mgt/mgt_pool.c

```diff
@@ -57,6 +57,8 @@ static int
 tweak_thread_pool_min(struct vsb *vsb, const struct parspec *par,
     const char *arg)
 {
+	MCF_ParamConf(MCF_MINIMUM, "thread_pool_min",
+	    "%u", 5 /* TASK_QUEUE_END */);
```

Comment:
If we hard-code the minimum, I don't think it belongs here, but in the param spec instead.

Reply:
the field is a `char *`, so a literal "5" triggers a compiler error

Reply:
sorry, that's the dyn_min.
/me bad

Reply:
I don't understand, I'm talking about the `min` field here:

https://github.com/varnishcache/varnish-cache/blob/c59b80661bf4f331e908ce4e9f1e85038c929829/bin/varnishd/mgt/mgt_param.h#L38-L44

Reply:
I posted before seeing your reply, I'll let you fix this.

Reply:
fixed, thank you